src/lib/config.js

```diff
@@ -335,4 +378,67 @@ function getExplorerName(graphQLUrl) {
     .filter((item) => item === 'minascan' || item === 'minaexplorer')?.[0];
 }
 
+async function printDeployAliasesConfig(deployAliasesConfig) {
+  // Build table of existing deploy aliases found in their config.json
+  let tableData = [
+    [chalk.bold('Name'), chalk.bold('URL'), chalk.bold('Smart Contract')],
+  ];
+  for (const deployAliasName in deployAliasesConfig.deployAliases) {
+    const { url, smartContract } =
+      deployAliasesConfig.deployAliases[deployAliasName];
+    tableData.push([
+      deployAliasName,
+      url ?? '',
+      smartContract ?? chalk.gray('(never deployed)'),
+    ]);
+  }
+  // Sort alphabetically by deploy alias name.
+  tableData = tableData.sort((a, b) => a[0].localeCompare(b[0]));
+  const tableConfig = {
+    border: getBorderCharacters('norc'),
+    header: {
+      alignment: 'center',
+      content: chalk.bold('Deploy aliases in config.json'),
+    },
+  };
+  // Show "none found", if no deploy aliases exist.
+  if (tableData.length === 1) {
+    // Add some padding to empty name & url columns, to feel more natural.
+    tableData[0][0] = tableData[0][0] + ' '.repeat(2);
+    tableData[0][1] = tableData[0][1] + ' '.repeat(3);
+
+    tableData.push([[chalk.gray('None found')], [], []]);
+    tableConfig.spanningCells = [{ col: 0, row: 1, colSpan: 3 }];
+  }
+  // Print the table. Indented by 2 spaces for alignment in terminal.
+  const msg = '\n  ' + table(tableData, tableConfig).replaceAll('\n', '\n  ');
+  log(msg);
+}
+
+function printInteractiveDeployAliasConfigSuccessMessage(
+  deployAliasesConfig,
+  deployAliasName,
+  feepayerKeyPair
+) {
+  const explorerName = getExplorerName(
+    deployAliasesConfig.deployAliases[deployAliasName]?.url
+  );
+  const str =
+    `\nSuccess!\n` +
+    `\nNext steps:` +
+    `\n  - If this is the testnet, request tMINA at:\n    https://faucet.minaprotocol.com/?address=${encodeURIComponent(
```

Comment:
If we have an alias using lightnet, could we remove the line about requesting tMINA at the faucet?

Reply:
This one is for regular (interactive) deploy alias configuration.